docs/getting-started/architecture-and-components/offline-store.md

```diff
@@ -13,3 +13,5 @@ It is not possible to query all data sources from all offline stores, and only a
 
 Please see the [Offline Stores](../../reference/offline-stores/) reference for more details on configuring offline stores.
 
+Please see the [Push Documentation](reference/data-sources/push.md) for reference on how to push features directly to the offline store in your feature store.
```

Comment:
nit: maybe rename to `Push Source` instead of `Push Documentation`?

Reply:
Done